ui_base/src/main/java/ua/gov/diia/ui_base/fragments/BaseComposeFragment.kt

```diff
@@ -0,0 +1,40 @@
+package ua.gov.diia.ui_base.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModel
+
+abstract class BaseComposeFragment<VM : ViewModel> : Fragment() {
+
+    protected abstract val viewModel: VM
+
+    protected open fun buildViewModel(viewModel: VM) {}
+
+    final override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        buildViewModel(viewModel)
+    }
+
+    @Composable
+    protected abstract fun Content(viewModel: VM)
+
+    final override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View = ComposeView(requireContext()).apply {
+        setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+
+        setContent { Content(viewModel) }
+    }
+
+    final override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+    }
+}
```

Comment:
add new line please

Reply:
it's only github stuff, it makes no sense fot the code

<img width="135" alt="Screenshot 2024-03-14 at 10 47 01" src="https://github.com/diia-open-source/android-diia/assets/18163868/25b0bfb5-00fe-40a9-ac88-f23cf1d6dc2a">